404.html

```diff
@@ -18,6 +18,11 @@
 </head>
 
 <body>
+    <div class="cover">
+        <center><img src="coding.png" width="275dp"></center>
```

Comment:
This image doesn't exist. Did you forget to upload it?

Reply:
is it OK now?